cmd/client/capture.go

```diff
@@ -71,6 +71,10 @@ var CaptureCreate = &cobra.Command{
 		}
 	},
 	Run: func(cmd *cobra.Command, args []string) {
+		if !config.GetConfig().GetBool("analyzer.capture_enabled") {
```

Comment:
I don't think we should put this test here as it depends on the config file read by the client and the config of the client could allow the capture creation. Check on the API side should be enough